### PR TITLE
git-merge-*: add page

### PR DESCRIPTION
**Pull request by pg99696**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/git-merge-into.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# git merge-into
+
+> Merge one branch into another branch.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-merge-into>.
+
+- Merge source branch into destination branch:
```
Comment (EmilySeville7cfg) [inline pages/common/git-merge-into.md:7]:
```suggestion
- Merge a source branch into a specific destination branch:
```
Optional change: no such style guide requirement exists.

File: pages/common/git-merge-into.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+`git merge-into {{source-branch}} {{destination-branch}}`
```
Comment (EmilySeville7cfg) [inline pages/common/git-merge-into.md:9]:
```suggestion
`git merge-into {{source_branch}} {{destination_branch}}`
```

Mandatory change: snake case is [required](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming).

File: pages/common/git-merge-into.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+- Merge current branch into destination branch:
```
Comment (EmilySeville7cfg) [inline pages/common/git-merge-into.md:11]:
```suggestion
- Merge current branch into a specific destination branch:
```

Optional change: no such style guide requirement exists.

File: pages/common/git-merge-into.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+`git merge-into {{destination-branch}}`
```
Comment (EmilySeville7cfg) [inline pages/common/git-merge-into.md:13]:
```suggestion
`git merge-into {{destination_branch}}`
```

Mandatory change: snake case is [required](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming).

File: pages/common/git-merge-repo.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# git merge-repo
+
+> Merge two repository histories.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-merge-repo>.
+
+- Merge repository's branch into current repository's directory:
```
Comment (EmilySeville7cfg) [inline pages/common/git-merge-repo.md:7]:
```suggestion
- Merge a repository's branch into the current repository's directory:
```

Optional change: no such style guide requirement exists.

File: pages/common/git-merge-repo.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+`git merge-repo {{local-path-to-repo.git}} {{branch-name}} {{path/to/directory}}`
+
+- Merge remote repository's branch into current repository's directory, not preserving history:
```
Comment (EmilySeville7cfg) [inline pages/common/git-merge-repo.md:11]:
```suggestion
- Merge a remote repository's branch into the current repository's directory, not preserving history:
```
Optional change: no such style guide requirement exists.

File: pages/common/git-merge-repo.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# git merge-repo
+
+> Merge two repository histories.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-merge-repo>.
+
+- Merge repository's branch into current repository's directory:
+
+`git merge-repo {{local-path-to-repo.git}} {{branch-name}} {{path/to/directory}}`
```
Comment (EmilySeville7cfg) [inline pages/common/git-merge-repo.md:9]:
```suggestion
`git merge-repo {{local_path_to_repo.git}} {{branch_name}} {{path/to/directory}}`
```
Mandatory change: snake case is [required](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming).

```suggestion
`git merge-repo {{local/path/to/repo.git}} {{branch_name}} {{path/to/directory}}`
```

Optional change: no such style guide requirement exists.

File: pages/common/git-merge-repo.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+
+- Merge remote repository's branch into current repository's directory, not preserving history:
+
+`git merge-repo {{remote-path-to-repo}} {{branch-name}} .`
```
Comment (EmilySeville7cfg) [inline pages/common/git-merge-repo.md:13]:
```suggestion
`git merge-repo {{remote_path_to_repo}} {{branch-name}} .`
```

Mandatory change: snake case is [required](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#naming).

```suggestion
`git merge-repo {{remote/path/to/repo}} {{branch-name}} .`
```

Optional change: no such style guide requirement exists.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
- Changes are requested because some fixes are mandatory.
- Thanks for a contribution. ❤️

**Review by kbdharun — APPROVED** [review]
LGTM, after @EmilySeville7cfg's suggestions, are addressed.

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by pg99696** [comment]
@kbdharun / @EmilySeville7cfg , Please review all the changes once again and let me know if anything is missed. Please close the PR once reviewed.

**Review by kbdharun — APPROVED** [review]

**Comment by pg99696** [comment]
Thanking you @kbdharun and @EmilySeville7cfg for the quick review and your valuable comments.

**Pull request merged**